README.md

```diff
@@ -56,53 +56,65 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION  -->
 
-# 📖 Webpack template <a name="about-project"></a>
+# 📖 Smart Toto List <a name="about-project"></a>
 
-**Webpack template** write your description here.
+**Smart Toto List** is a powerful yet simple web-based application designed to help you manage your daily tasks and stay on top of your schedule. With its user-friendly interface, you can easily add, remove, and mark tasks as complete, as well as remove all completed tasks with just a click of a button. 
+
+The application has been designed to be easy to use, with a clean and intuitive interface that allows you to quickly add new tasks and manage existing ones. Whether you're a busy professional, a student, or a stay-at-home parent, Smart Todo List makes it easy to keep track of your tasks and stay organized.
+
+Overall, Smart Todo List is a versatile tool that can help you stay organized and focused. It's fully responsive, allowing you to manage your tasks from any device with an internet connection. Keep your to-do list up-to-date and clutter-free with Smart Todo List.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
-- Write tech here
+- HTML/CSS/JavaScript
 
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
-- List features here
+- **Add tasks:** Easily add new tasks to your to-do list with a simple input field.
+- **Remove tasks:** Remove tasks that are no longer relevant or necessary.
+- **Mark tasks as complete:** Check off tasks that you've completed to help you stay organized.
+- **Remove completed tasks:** Remove all completed tasks with the click of a button to keep your list clutter-free.
+- **Simple and user-friendly interface:** Designed to be easy to use and navigate, with an intuitive interface that's accessible to anyone.
+- **Fully responsive:** Access your to-do list from any device with an internet connection, including smartphones, tablets, and laptops.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO  -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-- Click <a href="https://link-to-live-demo">here</a> to open the live demo.
+- Click <a href="https://wacoo.github.io/smart-todo-list/dist/index.html">here</a> to open the live demo.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 💻 Project Presentation <a name="presentation"></a>
-- Click <a href="link-here">here</a> to open the presentation.
+<!-- - Click <a href="link-here">here</a> to open the presentation. -->
+- Comming soon
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 <!-- GETTING STARTED -->
```

Comment:
- [ ] Your `Live Demo ` and `Project Presentation` links currently lead to error pages. So I will suggest that you either remove these two sections completely from your README or replace the false links with `N/A` or `Coming Soon` so as to not mislead your users or visitors :wink: 

- [ ] Your MIT license link currently leads to an error page. I have attached a screenshot below. I think this might be because you don't have a license file in your project. So I will suggest that you include an MIT license file in your project. You can use this  [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)  to learn how to add one to your project :wink:

![image](https://github.com/wacoo/smart-todo-list/assets/102699958/e756637b-b45f-474b-9c8e-aef5ce5f4de8)